src/openrct2/drawing/string.c

```diff
@@ -739,7 +739,7 @@ uint8 findClosestIndex(uint8 red, uint8 green, uint8 blue)
     return closest;
 }
 
-uint8 blend(const uint8 base, const uint8 layer)
```

Comment:
Naming could probably be improved upon. As well as pointing out that the values are palette indexes.

Reply:
Agreed, it's probably not clear what the purpose of a 'blend' function in string.c. I was thinking it might be better to export these functions from colour.{c,h} instead, and move the blend table there as well. What are your thoughts on that?